src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1333,36 +1336,38 @@ internal static bool IsCustomTaskType(this NamedTypeSymbol type, out NamedTypeSy
         /// </summary>
         internal static TypeSymbol NormalizeTaskTypes(this TypeSymbol type, CSharpCompilation compilation)
         {
-            NormalizeTaskTypesCore(compilation, ref type);
+            NormalizeTaskTypesInType(compilation, ref type);
             return type;
         }
 
         /// <summary>
         /// Replace Task-like types with Task types. Returns true if there were changes.
         /// </summary>
-        private static bool NormalizeTaskTypesCore(CSharpCompilation compilation, ref TypeSymbol type)
+        private static bool NormalizeTaskTypesInType(CSharpCompilation compilation, ref TypeSymbol type)
         {
             switch (type.Kind)
```

Comment:
Neither an error type or pointer can be task-like though.  What would be the advantage of handling them here? 

---
In reply to: [74285338](https://github.com/dotnet/roslyn/pull/12671#discussion_r74285338) [](ancestors = 74285338)